target-app/Onboarding/Views/SignUpViewController.swift

```diff
@@ -0,0 +1,224 @@
+//
+//  SignUpViewController.swift
+//  target-app
+//
+//  Created by Andres Leonel Torrico Cossio on 26/08/2022.
+//
+
+import UIKit
+
+class SignUpViewController: UIViewController {
+
+    private lazy var overlayImageView = UIImageView()
+    
+    private lazy var titleLabel = UILabel(style: .primary(text: "signin_title".localized))
+    
+    private lazy var nameLabel = UILabel(style: .secondary(text: "NAME"))
+    
+    private lazy var nameField = UITextField(
+        target: self
+    )
+
+    private lazy var emailLabel = UILabel(style: .secondary(text: "signin_email_label".localized))
+    
+    private lazy var emailField = UITextField(
+        target: self
+    )
+    
+    private lazy var passwordLabel = UILabel(style: .secondary(text: "signin_password_label".localized))
+    
+    private lazy var passwordField = UITextField(
+        target: self,
+        placeholder: "MIN. 6 CHARACTERS LONG"
+    )
+    
+    private lazy var confirmPasswordLabel = UILabel(style: .secondary(text: "CONFIRM PASSWORD"))
+    
+    private lazy var confirmPasswordField = UITextField(
+        target: self
+    )
+    
+    private lazy var genderLabel = UILabel(style: .secondary(text: "GENDER"))
+    
+    private lazy var genderField = UITextField(
+        target: self,
+        placeholder: "SELECT YOUR GENDER"
+    )
+    
+    private lazy var signInButton = UIButton(
+        style: .secondary(title: "signin_button_text".localized)
+    )
+
+    private lazy var lineView = UIView()
+    
+    private lazy var signUpButton = UIButton(
+        style: .primary(title: "signup_button_text".localized)
+    )
+    
+    lazy var scrollView: UIScrollView = {
```

Comment:
There is some reason to not let this variables as `private`?

Reply:
You’re right! Will fix it in next PR's